### PR TITLE
KOGITO-5525 Add protobuf marshalling related resources to native image

### PR DESCRIPTION
**Pull request by cristianonicolai**
Assembly:
- https://github.com/kiegroup/kogito-examples/pull/803
- https://github.com/kiegroup/kogito-runtimes/pull/1460

### CODE REVIEW

**Comment by cristianonicolai** [comment]
Jenkins rerun apps tests

**Comment by radtriste** [comment]
I guess we are going to need https://github.com/kiegroup/kogito-apps/pull/951 ...

**Comment by cristianonicolai** [comment]
Jenkins rerun apps tests

File: quarkus/extensions/kogito-quarkus-processes-extension/kogito-quarkus-processes-deployment/src/main/java/org/kie/kogito/quarkus/processes/deployment/ProcessesAssetsProcessor.java [hunk 1]
```diff
@@ -195,12 +221,9 @@ public void postGenerationProcessing(
...
 
-        if (persistenceGenerator.persistenceType().equals(PersistenceGenerator.MONGODB_PERSISTENCE_TYPE)) {
-            addInnerClasses(KogitoProcessInstanceProtobuf.class, reflectiveClass);
-            addInnerClasses(KogitoTypesProtobuf.class, reflectiveClass);
-            addInnerClasses(KogitoNodeInstanceContentsProtobuf.class, reflectiveClass);
-            addInnerClasses(KogitoWorkItemsProtobuf.class, reflectiveClass);
-            reflectiveClass.produce(new ReflectiveClassBuildItem(true, true, "java.lang.String"));
+        if (persistenceGenerator.persistenceType().equals(PersistenceGenerator.POSTGRESQL_PERSISTENCE_TYPE) ||
+                persistenceGenerator.persistenceType().equals(PersistenceGenerator.JDBC_PERSISTENCE_TYPE)) {
+            resourcePatterns.produce(new NativeImageResourcePatternsBuildItem.Builder().includeGlob("sql/*.sql").build());
```
Comment (cristianonicolai) [inline quarkus/extensions/kogito-quarkus-processes-extension/kogito-quarkus-processes-deployment/src/main/java/org/kie/kogito/quarkus/processes/deployment/ProcessesAssetsProcessor.java:226]: @tiagodolphine hopefully fixed here

**Comment by radtriste** [comment]
jenkins rerun apps tests

**Comment by cristianonicolai** [comment]
@radtriste @tiagodolphine all good to proceed with this one?

**Review by radtriste — APPROVED** [review]

**Comment by cristianonicolai** [comment]
Jenkins rerun examples tests

**Review by tiagodolphine — APPROVED** [review]

**Comment by cristianonicolai** [comment]
Jenkins rerun examples tests

**Comment by cristianonicolai** [comment]
Jenkins rerun examples tests

**Comment by cristianonicolai** [comment]
Jenkins rerun optaplanner tests

**Pull request merged**